src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -209,12 +209,17 @@ private string BuildLocalApp([CallerMemberName] string testName = "TestName", st
         return publishDirectory;
     }
 
-    [DockerAvailableTheory(Skip = "https://github.com/dotnet/sdk/issues/33858")]
-    [InlineData(false)]
-    [InlineData(true)]
-    public async Task EndToEnd_NoAPI_Web(bool addPackageReference)
+
+    [DockerAvailableTheory()]
+    [InlineData("webapi", false)]
+    [InlineData("webapi", true)]
+    [InlineData("worker", false)]
+    [InlineData("worker", true)]
+    [InlineData("console", false)]
+    [InlineData("console", true)]
```

Comment:
These are our positive test cases for container publish. We should have negative test cases for test projects and library projects, but perhaps in the Publish SDK tests directly? @vijayrkn any good ideas where those should live?